fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/POSITION/MfdFmsPositionNavaids.tsx

```diff
@@ -29,7 +29,7 @@ interface SelectedNavaid {
   class: Subject<string>;
 }
 
-const NAVAID_TYPE_STRINGS: Record<SelectedNavaidType, string> = {
+export const NAVAID_TYPE_STRINGS: Record<SelectedNavaidType, string> = {
```

Comment:
I don't think individual pages should export stuff like this, and doing so increases chances of circular deps. Any shared things belong elsewhere.